frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionsCreate.js

```diff
@@ -822,12 +822,21 @@ const SubmissionsCreate = ({ path }) => {
         const billPaymentStatus = await openPaymentPortal(bill);
         setPaymentStatus(billPaymentStatus);
         await applicationRefetch();
-        console.log(billPaymentStatus);
         if (billPaymentStatus === true) {
           setMakePaymentLabel(false);
           setShowPaymentModal(false);
           setShowSuccessModal(true);
+
           createPendingTask({ name: t("MAKE_PAYMENT_SUBMISSION"), status: "MAKE_PAYMENT_SUBMISSION", isCompleted: true });
+
+          if (applicationDetails?.additionalDetails?.isResponseRequired === true) {
+            createPendingTask({
+              name: t("RESPOND_TO_PRODUCTION_DOCUMENTS"),
+              status: "RESPOND_TO_PRODUCTION_DOCUMENTS",
+              stateSla: orderDetails?.orderDetails?.dates?.submissionDeadlineDate,
+              assignees: applicationDetails?.additionalDetails?.respondingParty?.map((item) => ({ uuid: item?.uuid[0] })),
```

Comment:
Don't map first only first uuid. respondingParty will have list of uuid, map it here

Reply:
done